### PR TITLE
Юферев -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by YuferevKirill**

### CODE REVIEW

**Comment by YuferevKirill** [comment]
Юферев -- Лабораторная работа #0

**Comment by YuferevKirill** [comment]
@Nikita-Mitrokhin, @mkildishev, Could you review tests?

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertFalse(complexNumber.equals(24));
+    }
+
+    @Test
+    public void canFormatToString() {
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertEquals("2.4 + 4.4i", complexNumber.toString());
+    }
+
```
Comment (Nikita-Mitrokhin) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java:24]: Expected value should not be a magical constant, create a dedicated variable.
it should look something like this: 
        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
        var expectedString = "2.4 + 4.4i";
        assertEquals(expectedString, complexNumber.toString());

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public void canGetImaginaryPart() {
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertTrue(complexNumber.getImaginary() == 4.4);
+    }
+
+    @Test
+    public void canGetRealPart() {
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertEquals(2.4, complexNumber.getReal(), delta);
+    }
```
Comment (Nikita-Mitrokhin) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java:37]: Similarly here:
        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
        var expectedRealPart = 2.4;
        assertEquals(expectedRealPart, complexNumber.getReal(), delta);

**Comment by mkildishev** [comment]
Looks good, 👍

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java (hunk 1 above)
Comment (YuferevKirill) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java:24]: fixed

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java (hunk 2 above)
Comment (YuferevKirill) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java:37]: fixed too, thx for review

**Comment by Nikita-Mitrokhin** [comment]
:+1:

**Comment by YuferevKirill** [comment]
@kirill-korniakov

**Pull request merged**